packages/polar/src/lib/compile/checkEnv.ts

```diff
@@ -53,22 +53,26 @@ export function checkEnv (
 
   const wasmInstalled: boolean = getWebAssemblyInstalled();
 
-  if (!rustcCurrVersion || semver.lt(rustcCurrVersion, rustcVersion)) {
+  if (!rustcCurrVersion) {
+    console.log(`Error: rustc not installed.`);
+    return false;
+  } else if (!rustcCurrVersion || semver.lt(rustcCurrVersion, rustcVersion)) {
```

Comment:
version not present is already checked above

```suggestion
  } else if (semver.lt(rustcCurrVersion, rustcVersion)) {
```